app/javascript/react/screens/App/Overview/Overview.js

```diff
@@ -273,11 +279,20 @@ class Overview extends React.Component {
               redirectTo={this.redirectTo}
             />
           )}
-          <InfrastructureMappingsList
-            clusters={clusters}
-            transformationMappings={transformationMappings}
-            createInfraMappingClick={showMappingWizardAction}
-          />
+          {hasSufficientProviders ? (
+            <InfrastructureMappingsList
+              clusters={clusters}
+              transformationMappings={transformationMappings}
+              createInfraMappingClick={showMappingWizardAction}
+            />
+          ) : (
+            <OverviewEmptyState
+              description="The VMWare and Red Hat Virtualization providers must be configured before attempting a migration."
```

Comment:
for i18n , you can add `{__("The Vmware...")}`

Reply:
Good catch, it was because I copied and pasted another OverviewEmptyState that wasn't i18n-ized.. i'll fix those ones real quick